content/websocket-sample.md

```diff
@@ -0,0 +1,190 @@
++++
+title = "Fission WebSocket Sample"
+date = "2021-06-11T10:50:51+05:30"
+author = "Gaurav Gahlot"
+description = "Write a serverless broadcast function with NodeJS"
+categories = ["Fission", "Serverless", "Application"]
+type = "post"
++++
+
+## Introduction
+
+In this post we will be into how we can develop a simple chat application using Fission functions.
+Fission's NodeJS environment now has built in support for [WebSocket][1].
```

Comment:
Instead of pointing to [1] we can use a format like this where the text becomes clickable:

```suggestion
Fission's NodeJS environment now has built in support for [WebSocket](https://www.npmjs.com/package/websocket).
```

Reply:
Also probably instead if NodeJS's package, we should point to websocket protocol info page?

Reply:
- The text is clickable now as well. 
- Should we point to this doc? https://datatracker.ietf.org/doc/html/rfc6455

Reply:
Oh wow, TIL!